apps/src/lib/util/mlApi.js

```diff
@@ -0,0 +1,32 @@
+/* global Promise */
+
+import {predict} from '@cdo/apps/MLTrainers';
+
+export const commands = {
+  async getPrediction(opts) {
+    return new Promise((resolve, reject) => {
+      $.ajax({
+        url: '/api/v1/ml_models/names',
+        method: 'GET'
+      })
+        .then(data => {
+          const modelId = data.find(model => model.name === opts.model).id;
```

Comment:
I suspect there will be a way to avoid making this call here since we should already know the model id in the full flow, but I haven't sussed that out quite yet. Maybe something kind of like a "hidden" parameter passed to the predict block? Still puzzling.